server/service/src/preference/upsert.rs

```diff
@@ -176,151 +176,89 @@ pub fn upsert_preferences(
 
             // Store preferences, input could be array of store IDs and values - iterate and insert...
             if let Some(inputs) = manage_vaccines_in_doses_input {
-                upsert_store_input(connection, manage_vaccines_in_doses, inputs)?
+                upsert_store_input(connection, manage_vaccines_in_doses, inputs)?;
```

Comment:
huh wondering how these semi colons weren't there when I added `upsert_store_input` or at least seems like a weird thing for the git merge to drop 🤔

Reply:
Also in retrospect I kinda feel I should have put the "if let" inside the function too as it's repeated for all of these Options... It's the sort of thing I'd flip flop on though 😆